discord/member.py

```diff
@@ -1148,3 +1158,17 @@ def is_timed_out(self) -> bool:
         if self.timed_out_until is not None:
             return utils.utcnow() < self.timed_out_until
         return False
+
+    def has_unusual_dm_activity(self) -> bool:
```

Comment:
I don't really like this method name, not a fan of `has_` methods unless it's a last resort kind of thing.